modules/quick_install.py

```diff
@@ -298,7 +298,7 @@ def docker_setup(log_file, config_path="/etc/ideam/ideam.conf"):
 
     output_info("Starting Catalogue quick install")
     subprocess.call('tasks/hypercat/quick-catalogue.sh ' + str(
-        subprocess.check_output("docker port catalogue | grep 22 | cut -d : -f 2", shell=True)).strip(),
+        subprocess.check_output("docker port hypercat | grep 22 | cut -d : -f 2", shell=True)).strip(),
```

Comment:
Hey Poorna @pct960 ,
It would be better to change the container name from `hypercat` to `catalogue`

Reply:
I initially went on to do that. However, there is some problem with the war file compilation on my system. So, I had to use the original war file which has ``hypercat`` and not ``catalogue``. I will eventually change this, once I fix the eclipse issue on my system.